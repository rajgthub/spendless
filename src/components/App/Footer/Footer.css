.footer{
    /* position: fixed;
    bottom: 0; */
    height: 15vh;
    width: 100%;
    background-color: black;
    color: white;
    text-align: center;
    padding: 1rem;
}