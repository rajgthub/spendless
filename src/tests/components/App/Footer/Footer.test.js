import React from "react";
import ReactShallowRenderer from "react-test-renderer/shallow";
import Footer from "../../../../components/App/Footer";
test("should render Header ", () => {
  const renderer = new ReactShallowRenderer();
  renderer.render(<Footer />);
  expect(renderer.getRenderOutput()).toMatchSnapshot();
});
