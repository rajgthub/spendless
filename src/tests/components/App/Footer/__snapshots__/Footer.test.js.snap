// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header  1`] = `
<div
  className="footer"
>
  <h1>
    @
  </h1>
</div>
`;
